megamek/src/megamek/common/AmmoType.java

```diff
@@ -10587,38 +10587,6 @@ private static AmmoType createISNailRivetGunAmmoHalf() {
         return ammo;
     }
 
-    /*
-     * private static AmmoType createCLNailRivetGunAmmo() { AmmoType ammo = new
```

Comment:
I think we keep these around in case they get implemented. Unless there's no such thing as clan nail rivet gun ammo or clan anil rivet gun half ammo, in which case, never mind.

Reply:
It hasn't been used since MM was on GitHub, and the text predates that.

It's also just not useful to retain ancient code just in case...